packages/forms-web-app/src/views/full-appeal/submit-appeal/letter-confirming-application.njk

```diff
@@ -0,0 +1,79 @@
+{% extends "layouts/main.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set labelText = "Drag and drop or choose a file" %}
+{% set title = "The letter confirming your planning application - Appeal a planning decision - GOV.UK" %}
+{% block pageTitle %}{{ "Error: " + title if errors else title }}{% endblock %}
+
+{% block backButton %}
+  {{ govukBackLink({
+    text: 'Back',
+    href: '/full-appeal/submit-appeal/task-list',
```

Comment:
is this backlink correct? scenario 10 suggests it should return to the design and access statement page

Reply:
resolved.